model/database/spans.yaml

```
@@ -179,9 +205,16 @@ groups:
         requirement_level:
           conditionally_required: If available.
         note: ""  # overriding the base note
+      - ref: db.response.status_code
+        brief: >
+          The HTTP response code returned by the Couch DB.
+        examples: ["200", "201", "429"]
+        requirement_level:
+          conditionally_required: If response was received.
```

not sure if it's generally available to instrumentation, e.g. https://docs.couchbase.com/sdk-api/couchbase-java-client/com/couchbase/client/java/Collection.html#get(java.lang.String)

```suggestion
          conditionally_required: If response was received and the HTTP response code is available.
```